README.md

```diff
@@ -60,18 +59,12 @@ It can be used as a component of a frontend such as
 interface for wallets. Most users who would like to use Cardano should
 start with Daedalus.
 
-> :information_source: This source code repository contains the next major version of Cardano
-> Wallet, which has been completely rewritten for the
-> [Shelley](https://roadmap.cardano.org/) phase.
->
-> :bulb: The Byron version of Cardano Wallet is in the
-> [cardano-sl](https://github.com/input-output-hk/cardano-sl)
-> repository.
+## Quickstart
 
-## Getting Started
+The `cardano-wallet` executable is a HTTP server which manages your wallet(s). Here is one way to start the server using Docker: 
```

Comment:
I would propose:

```suggestion
The `cardano-wallet` application provides an HTTP server that you can use to manage your wallet(s). Here is one way to start the server using Docker: 
```

Reply:
I would be ok with your suggested rephrasing, but I do prefer the original one. My reasons are

* `executable` is more specific than `application`. In the context of the QuickStart, this distinction is useful — it confers that the focus of `cardano-wallet` is a single executable, not a bundle.
* `is an HTTP server` is again more specific than `provides an HTTP server` — it suggests that if you run `cardano-wallet` executable, then it will start listening on a port for HTTP requests. Again, this is useful information to be conferred in a QuickStart section, which should not only give commands to run the application, but also give a quick overview of what parts are involved — and this choice of verb subtly confers that.
* `which manages` is shorter than `that you can use to manage` without losing information. (In fact, the original suggests that `cardano-wallet` is now in charge of your wallets, not you). This shortening is a trick I picked up from [Nature's English Communication for Scientists][nature]:
    
    > A verb in the active voice does not require a person as the agent; an inanimate object is often appropriate.
    
    Put differently, making inanimate things do something, such as managing wallets, frequently improves readability.
    
    In addition, the word `which` confers that managing wallets is an essential part of `cardano-wallet`, not just an additional function `that` it can also do.

  [nature]: https://www.nature.com/scitable/ebooks/english-communication-for-scientists-14053993/126083980/